helix-core/src/test/java/org/apache/helix/integration/task/TestJobFailure.java

```diff
@@ -118,8 +118,15 @@ public void testNormalJobFailure(String comment, List<String> taskStates,
   }
 
   private Map<String, Map<String, String>> createPartitionConfig(List<String> taskStates,
-      List<String> expectedTaskEndingStates) {
+      List<String> expectedTaskEndingStates) throws Exception {
     Map<String, Map<String, String>> targetPartitionConfigs = new HashMap<>();
+    // Make sure external view is has been created for the resource
+    boolean isExternalViewCreated = TestHelper.verify(() -> {
+      ExternalView externalView =
+          _manager.getClusterManagmentTool().getResourceExternalView(CLUSTER_NAME, DB_NAME);
+      return (externalView != null);
+    }, TestHelper.WAIT_DURATION);
+    Assert.assertTrue(isExternalViewCreated);
```

Comment:
Nit, I would just do `assertTrue(TestHelper.verify());` to get rid of the temp boolean `isExternalViewCreated` var to minimize variable scope. Same for following usages.

Reply:
Good suggestion. Changed.